docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+   app:
+       build:
+           context: .
+       ports:
+           - "8001:8000"
+       volumes:
+           - ./:/app
+           - my_media:/files/media
+       command: >
+           sh -c "python manage.py wait_for_db &&
+                  python manage.py migrate &&
+                  python manage.py runserver 0.0.0.0:8000"
+       env_file:
+           - .env
+       depends_on:
+           - db
+
+   db:
+       image: postgres:14-alpine
+       restart: always
+       ports:
+           - "5433:5432"
```

Comment:
The `ports` configuration for the `db` service maps port `5433` on the host to `5432` on the container. Ensure that this is intentional and aligns with your application's requirements.